src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
 
 import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = Comment;
```

Comment:
Props it's separate object
You should pass comment in some key in this object
```suggestion
type Props = {
  comment: Comment;
};
```